### ISSUE TITLE
Describe carousel design challenges in greater detail

### ISSUE THREAD

**Issue by yatil** [w3c/wai-tutorials#165]
**Original Comment:**

> Should describe more about why they are problems (distracting motion, might move too quickly, lack of keyboard control...). That is preparatory to describing the benefits of following the guidance in "Why is this important".
